contrib/kind.sh

```diff
@@ -65,41 +80,108 @@ print_params()
      echo "KIND_HA = $KIND_HA"
      echo "KIND_CONFIG_FILE = $KIND_CONFIG "
      echo "KIND_REMOVE_TAINT = $KIND_REMOVE_TAINT"
+     echo "KIND_IPV4_SUPPORT = $KIND_IPV4_SUPPORT"
+     echo "KIND_IPV6_SUPPORT = $KIND_IPV6_SUPPORT"
      echo ""
 }
 
 parse_args $*
 
-MASTER_COUNT=1
+# Set default values
+KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-ovn}
 K8S_VERSION=${K8S_VERSION:-v1.17.2}
 KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS:-false}
 KIND_HA=${KIND_HA:-false}
-if [ "$KIND_HA" == true ]; then
-  DEFAULT_KIND_CONFIG=./kind-ha.yaml
-  MASTER_COUNT=`grep -c "^\s-\srole\s*:\s*control-plane" kind-ha.yaml`
-else
-  DEFAULT_KIND_CONFIG=./kind.yaml
-  MASTER_COUNT=`grep -c "^\s-\srole\s*:\s*control-plane" kind.yaml`
-fi
-KIND_CONFIG=${KIND_CONFIG:-$DEFAULT_KIND_CONFIG}
+KIND_CONFIG=${KIND_CONFIG:-./kind.yaml.j2}
 KIND_REMOVE_TAINT=${KIND_REMOVE_TAINT:-true}
+KIND_IPV4_SUPPORT=${KIND_IPV4_SUPPORT:-true}
+KIND_IPV6_SUPPORT=${KIND_IPV6_SUPPORT:-false}
+
+# Input not currently validated. Modify outside script at your own risk.
```

Comment:
We have defaulted these subnets in KIND, just FYI
https://github.com/kubernetes-sigs/kind/blob/master/pkg/apis/config/v1alpha4/default.go